scripts/get_ek.sh

```diff
@@ -39,20 +46,23 @@ if [ -n "$TPM_VER_1_2" ]; then
     readCmd="tpm_nvread ""$TPM1_AUTH_SETTINGS"" ""$indexCmd"" -s %s -n %s | sed -r \"s/[0-9a-f]+ ([ 0-9a-f]{48}).*/\\\\1/\" | tr -d [[:space:]]"
     nvBufferedRead="1"
 elif [ -n "$TPM_VER_2_0" ]; then 
-    TPM2_TOOLS_VER_1=$(yum list installed tpm2-tools | grep --quiet -E "[ \t]+1\." && echo "1" || echo "") # Figure out best way to determine TPM2_TOOLS version.  query yum?
-    TPM2_TOOLS_VER_2=$(yum list installed tpm2-tools | grep --quiet -E "[ \t]+2\." && echo "1" || echo "") # Figure out best way to determine TPM2_TOOLS version.  query yum?
-    TPM2_TOOLS_VER_3=$(yum list installed tpm2-tools | grep --quiet -E "[ \t]+[3-9]+\." && echo "1" || echo "") # Can't base this on specific minor versions.  Major version will suffice
+    TPM2_TOOLS_VER_1=$("$distCmd" list installed tpm2-tools 2> /dev/null | grep --quiet -E "[ \t]+1\." && echo "1" || echo "") # Figure out best way to determine TPM2_TOOLS version.  query yum/apt?
```

Comment:
```apt list installed``` also outputs warning messages that the format is not consistent.  For simplicity I am redirecting that warning until I find another way to detect the tpm2-tools version.